wie-geht-studium/lehrbuecher.tex

```diff
@@ -81,7 +80,76 @@ \subsubsection{Mathematische Methoden}
 		In diesem Buch ist die Mathe, die man in den ersten beiden Semestern eines Physikstudiums braucht, anschaulich und ausführlich erklärt. Dabei wird bewusst auf mathematische Beweise verzichtet und mehr auf die physikalische Interpretation eingegangen. Sehr gut geeignet für Studis, die sich am Anfang in Theo ein wenig von der Mathe überrumpelt fühlen.}
 \end{description}
 
-\subsubsection*{Nachschlagewerke}
+\subsubsection{Informatik allgemein}
+
+In den Informatik-Vorlesungen wird allgemein sehr wenig bewusst Literatur verwendet,
+sondern hauptsächlich die Vorlesungsfolien und vor allem Online-Dokumentationen,
+insbesondere zu den Programmiersprachen, verwendet. Meistens werden gute Wikipedia-Artikel,
+Youtube-Tutorials, Stackoverflow-Posts und Google-Books bzw. Google-Scholar-Links näher angeschaut. 
+
+\subsubsection{Praktische Informatik und Programmierkurs}
+Für den Python-Programmierkurs sollte die Online-Dokumentation angeschaut werden:
```

Comment:
Der Programmierkurs ist in C++ und war nur einmal in Python der Teil sollte also raus